src/cellfinder_core/tools/array_operations.py

```diff
@@ -4,7 +4,7 @@
 
 
 # WARNING: skewed if does not fall exactly (integer multiple)
-def bin_array(arr, bin_width=15, bin_height=15):
+def bin_array(arr: np.ndarray, bin_width: int, bin_height: int):
```

Comment:
Are we removing the default values for width and height here intentionally? (Would make sense given that bin_width _should_ typically be a divisor of the array shape so it's hard to have a sensible default, but wanted to double-check?)

Reply:
Yes, removing defaults was deliberate. 👍 I've added a docstring